### ISSUE TITLE
Update gulp tasks

### ISSUE THREAD

**Issue by BurkusCat** [BurkusCat/r6-operator-counters#116]
- Gulp tasks no longer live reload properly, need to re-add it back in whilst keeping build non-live reload
- Scripts gulp task needs to not minify on gulp default and minify in gulp build

**Comment by abuDarda97** [comment]
Hey Burkus,

Can I work on this issue?

**Comment by CrayCrayDecay** [comment]
Hi Abu,

Feel free to work on this issue.

**Comment by abuDarda97** [comment]
Hi guys,

So, I npm installed gulp-livereload. Should I call the livereload function in every gulp file other than build.js?

**Comment by abuDarda97** [comment]
Hey guys,

So I was looking into the livereload issue. I think the problem might in the connect.js file, the way livereload is set up. I am not too sure what the actual problem is, I never used gulp before. When I added `.pipe(connect.reload());`  in the styles.js file, the livereload worked fine, when I changed something in neo4jd3.scss file the website updated.

**Comment by abuDarda97** [comment]
Hi @BurkusCat @CrayCrayDecay ,

I think I solved the issue the app works fine, so I didn't break anything. I just made minor adjustments to the gulp files. 

Thanks,

Abu

**Comment by anton-dealmeida** [comment]
Is this still open?

**Comment by BurkusCat** [comment]
Yes it is still open, a PR was raised and merged for this but then reverted because it caused a bug on the deployed site: https://github.com/BurkusCat/r6-operator-counters/pull/126

An issue was raised for a bug it caused https://github.com/BurkusCat/r6-operator-counters/issues/127

If you are interested in this issue, you may want to use the original PR as a reference but make sure the output of "gulp build" works as intended (that is what www.r6counters.com uses)
